readthedocs/builds/syncers.py

```diff
@@ -1,18 +1,22 @@
-"""Classes allowing copying files around.
+"""Classes to copy files between build and web servers
```

Comment:
Yup! I got a PR on ops, but it needs to be updated as it's conflicting with the other PR you opened around Syncers